project/api/management/commands/import_narratives.py

```diff
@@ -44,7 +37,6 @@ def handle(self, *args, **options):
             #     date = ceil(sum(gcal2jd(int(dates[0]), dates[1], 1))) + 0.0
             # else:
             #     date = ceil(sum(gcal2jd(int(dates[0]), dates[1], dates[2]))) + 0.0
-            print(row)
             # if sheet.title == "Russia":
             #     x_coord_split = row[2].value.split("°")
             #     y_coord_split = row[3].value.split("°")
```

Comment:
This bit is commented out because all the sheets had different date formats, we should decide how it's gonna be and finalized this.